sequencer-sqlite/Cargo.toml

```diff
@@ -15,24 +15,24 @@ sqlite-unbundled = ["sequencer/sqlite-unbundled"]
 embedded-db = ["sequencer/embedded-db"]
 
 [dependencies]
-alloy-node-bindings = "=0.12.5"
 sequencer = { path = "../sequencer" }
-tokio = { version = "1", default-features = false, features = [
```

Comment:
we obviously need these it's weird that locally compilation doesn't fail without them though

Reply:
Ah, I think it's because the binary isn't built without `--bin`.